api/tests/test_tasks_customer.py

```diff
@@ -0,0 +1,67 @@
+from django.test import TestCase
+from unittest.mock import patch, MagicMock
+
+from api.serializers import CustomerSerializer
+from api.tasks import check_customer_is_partner
+from sugarcrm.error import CrmAuthenticationError
+
+
+class TasksCustomerTests(TestCase):
+    def setUp(self):
+        self.data = {
+            "id": 1,
+            "date_created": "2021-05-06T13:06:18",
+            "date_created_gmt": "2021-05-06T11:06:18",
+            "date_modified": "2021-05-06T13:06:18",
+            "date_modified_gmt": "2021-05-06T11:06:18",
+            "email": "partner@email.com",
+            "first_name": "Test",
+            "last_name": "test",
+            "role": "customer",
+            "username": "test.test",
+            "billing": {
+                "first_name": "Test",
+                "last_name": "test",
+                "company": "",
+                "address_1": "sdadsad",
+                "address_2": "adsadsadasdadsa",
+                "city": "sadasdasdadsa",
+                "postcode": "33333",
+                "country": "ES",
+                "state": "B",
+                "email": "xxxx@xxxxx.com",
+                "phone": "4234324324234",
+            },
+            "shipping": {
+                "first_name": "Test",
+                "last_name": "test",
+                "company": "",
+                "address_1": "xx",
+                "address_2": "xx",
+                "city": "xx",
+                "postcode": "xx",
+                "country": "ES",
+                "state": "B",
+            },
+            "is_paying_customer": False,
+            "avatar_url": "http://1.gravatar.com/avatar/xx",
```

Comment:
do we need all these fields? IMO it's much better to provide only the ones that the code is going to need. Sometimes, I add one or two extra ones if they may help the reader understand what the JSON object is about. This makes the test smaller, to the point (no superfluous information) thus, more straightforward for the reader.